src/Shouldly/Configuration/DiffTool.cs

```diff
@@ -3,9 +3,25 @@
 using System.Diagnostics;
 using System.IO;
 using System.Linq;
+using System.Runtime.InteropServices;
 
 namespace Shouldly.Configuration
 {
+    public class DiffToolPath
+    {
+        public string Windows { private get; set; }
+
+        public string Mac { private get; set; }
+
+        public string TruePath
+        {
+            get
+            {
+                return RuntimeInformation.IsOSPlatform(OSPlatform.Windows) ? Windows : Mac;                
```

Comment:
Just a heads up, I didn't use this cause it isn't in lower versions of .net, you might want to conditionally compile this

Reply:
I thought that might be the case, let me know when it's ready for a look. I like it a lot so far 👍

Reply:
Did this in the end https://github.com/shouldly/shouldly/blob/9cd5d10c9a7a55d0a45a0daa091147333446178a/src/Shouldly/ShouldlyEnvironmentContext.cs